### ISSUE TITLE
static mode—no setup() / draw()—broken in the repo for Java2D

### ISSUE THREAD

**Issue by shiffman** [processing/processing#3130]
only tested retina mac os x 10.10.2.

Following code only draws black background:

```
size(200, 200);
background(0);
fill(255);
ellipse(100, 100, 100, 100);
```

**Comment by JobLeonard** [comment]
Not broken on Ubuntu Linux using 3.05a

**Comment by shiffman** [comment]
sorry, neglected to mention that I'm testing with the current build from source v0233.  I do not have this problem on 3.05a.

**Comment by JobLeonard** [comment]
Ah, ok. Will skip the inquiry then.

**Comment by joelmoniz** [comment]
I seem to be getting a 200x200 window with a white background when I run those 4 lines. I get the same result when I place those 4 lines in a ```setup()``` function. I'm testing on Windows 8.1.

On a side note (and I'm not very sure if this qualifies as a separate issue altogether or not), the following code does not produce any output at all (nor does it flag an error):

```
void setup() {
  size(200, 200);
  background(0);
  fill(255);
  ellipse(100, 100, 100, 100);
}

void draw() {
  background(0);
  fill(255);
  ellipse(100, 100, 100, 100);
}
```
  
while this code produces the desired output:  
  
```
void setup() {
  size(200, 200);
  background(0);
  fill(255);
}

void draw() {
  background(0);
  fill(255);
  ellipse(100, 100, 100, 100);
}
```

**Comment by dhawangupta** [comment]
I am using Processing 2.2.1 on Windows 8.1(x64).
All the aforementioned 3 pieces of code produced same output which is:
a 200x200 window with a black background and white circle with centre at 100,100.
So, there is no bug.

**Comment by benfry** [comment]
Ouch, that's a showstopper for 3.0a6.

**Comment by benfry** [comment]
Added the OpenGL version here: https://github.com/processing/processing/issues/3163

**Comment by benfry** [comment]
Fixed for 3.0a6.
